src/programs/BuddyProject.ts

```diff
@@ -381,3 +381,16 @@ export const cleanEntries = async (guild: Guild) => {
   let outputText = `Found ${unmatchedEntries[1]} unmatched members`;
   outputChannel.send(outputText);
 };
+
+export const checkAllEntries = async (guild: Guild) => {
+  const entries = await BuddyProjectEntryRepository();
+  const all = await entries.find({ where: { matched: false } });
+
+  const outputChannel = guild.channels.cache.find(
+    (c) => c.name == "buddy-project-matches"
+  ) as TextChannel;
+  const returnString = "";
+  all.forEach((each) => returnString + ` , ${each.user_id}`);
```

Comment:
```suggestion
  const returnString = all.map((each) => each.user_id).join(", ");
```
This is just a nice little touch; imo a little easier to read and doesn't produce a leading ` , `. Tbf, it is not really relevant because this will only be usable by us but figured, I'd still address it.

Reply:
Fixed